reporter-web-app/src/components/PackageFindingsTable.js

```diff
@@ -84,6 +84,17 @@ class PackageFindingsTable extends React.Component {
                 filters: (
                     () => Array.from(webAppPackage.detectedLicenses)
                         .map((license) => ({ text: license, value: license }))
+                        .sort(
+                            (a, b) => {
+                                if (a.text < b.text) {
+                                    return -1;
+                                }
+                                if (a.text > b.text) {
+                                    return 1;
+                                }
+                                return 0;
```

Comment:
What about this comment, @tsteenbe?

Reply:
Done

Reply:
Uhm, but shouldn't this just be something like `(a, b) => { return a.text.localeCompare(b.text) }` then?

Reply:
I was using `(a, b) => a.text.localeCompare(b.text)` until I noticed below warning on [MDN localeCompare](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare):

> Do NOT rely on exact return values of -1 or 1!
>
> Negative and positive integer results vary between browsers (as well as between browser versions) because the W3C specification only mandates negative and positive values. Some browsers may return -2 or 2, or even some other negative or positive value.

Reply:
Fixed

Reply:
Would be nice to use the simpler form already in this commit instead of fixing it up in the later commit.

Reply:
This still should be `.sort((a, b) => a.text.localeCompare(b.text))` already here.

Reply:
Sadly my comments were ignored :sob: